src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivClean.hpp

```diff
@@ -398,6 +376,32 @@ struct EvolutionMetavars {
   }
 };
 
+struct KerrHorizon {
+  using tags_to_observe =
+      tmpl::list<StrahlkorperTags::EuclideanSurfaceIntegralVector<
+          hydro::Tags::MassFlux<DataVector, 3, ::Frame::Inertial>,
+          ::Frame::Inertial>>;
+  using compute_items_on_source = tmpl::list<>;
+  using vars_to_interpolate_to_target =
+      tmpl::list<hydro::Tags::RestMassDensity<DataVector>,
+                 hydro::Tags::SpatialVelocity<DataVector, 3, Frame::Inertial>,
+                 hydro::Tags::LorentzFactor<DataVector>,
+                 gr::Tags::Lapse<DataVector>,
+                 gr::Tags::Shift<3, Frame::Inertial, DataVector>,
+                 gr::Tags::SqrtDetSpatialMetric<DataVector>>;
+  using compute_items_on_target = tmpl::push_front<
+      tags_to_observe,
+      StrahlkorperTags::EuclideanAreaElement<::Frame::Inertial>,
+      hydro::Tags::MassFluxCompute<DataVector, 3, ::Frame::Inertial>>;
+  using compute_target_points =
+      intrp::TargetPoints::KerrHorizon<KerrHorizon, ::Frame::Inertial>;
+  using post_interpolation_callback =
+      intrp::callbacks::ObserveTimeSeriesOnSurface<tags_to_observe, KerrHorizon,
+                                                   KerrHorizon>;
+  using interpolating_dg_component =
```

Comment:
Is InterpolationTargetTag now required to have this type alias?  Should be documented in InterpolationTarget.hpp near the top (including under what conditions it is required).

Reply:
this is only required if the interpolation target has `is_sequential` aliased to `std::false_type`. I'll add that documentation.